src/DynamoCoreWpf/UI/Prompts/DynamoMessageBox.xaml.cs

```diff
@@ -23,6 +27,15 @@ public partial class DynamoMessageBox : INotifyPropertyChanged
 
         #region Public Properties
         
+        [Obsolete("Do not use, instead use CustomDialogResult. This property will not be set during ShowDialog()")]
+        public new bool? DialogResult
+        {
+            get => base.DialogResult;
+            set => base.DialogResult = value;
+        }
+
+        internal MessageBoxResult CustomDialogResult { get; set; } = MessageBoxResult.None;
```

Comment:
is None equivalent to Cancel?
Do we handle None differently anywhere  ?
If not ...can we just use Cancel instead...might be simpler

Reply:
`None` occurs when the close button is clicked, Cancel only occurs when the `Cancel` button is clicked. In this PR I have modified our code to usually check for both of them as the same, but that may not always be the intended behavior.